compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1435,8 +1435,8 @@ error.cannot.update.constant.value=\
 error.cannot.assign.value.to.constant=\
   cannot assign a value to a constant
 
-error.cannot.define.constant.with.type=\
-  constant cannot be defined with type ''{0}'', expected a simple basic types or a map of a simple basic type
+error.invalid.const.declaration=\
+  type ''{0}'' is invalid in constant declaration
```

Comment:
Shall we have a bit more descriptive error message, something that says this type must be subtype of anydata and must not be never

Reply:
How about this? `type ''Foo'' is invalid in constant declaration, expected a simple basic type or a map of a simple basic type and must not be never`

Reply:
both maps and lists.
I prefer just saying `anydata`

Reply:
`type ''Foo'' is invalid in constant declaration, expected a subtype of anydata and must not be never` ?